File: quests/kodtaz/#Zoner1.pl
sub EVENT_SPAWN {
  	$x = $npc->GetX();
   	$y = $npc->GetY();
	quest::set_proximity($x - 10, $x + 10, $y - 10, $y + 10);
}

sub EVENT_ENTER {
$InInstanceIkky1 = quest::GetInstanceID("ikkinz",0);
  if(!defined $qglobals{ikkylockout0}) {
    if($InInstanceIkky1 > 0){
		quest::MovePCInstance(294,$InInstanceIkky1,-157,27,-2);
	}
        else {
		$client->Message(13, "You are not a part of an instance!");
	}
    }
        else {
	    quest::say("You have recently completed this trial, please come back at a later point");
	}
}

File: kodtaz/#Zoner1.lua
function event_spawn(e)
  local x = e.self:GetX()
  local y = e.self:GetY()
  eq.set_proximity(x - 10, x + 10, y - 10,  y + 10)
end

function event_enter(e)
  local dz_version = instance_version.ikkinz_chambers_of_singular_might
  e.other:MovePCDynamicZone("ikkinz", dz_version, false) -- Zone: ikkinz group 1 version
end
